src/LightAccount.sol

```diff
@@ -57,12 +57,14 @@ contract LightAccount is BaseAccount, TokenCallbackHandler, UUPSUpgradeable, Cus
         0x33e4b41198cc5b8053630ed667ea7c0c4c873f7fc8d9a478b5d7259cec0a4a00;
     // bytes4(keccak256("isValidSignature(bytes32,bytes)"))
     bytes4 internal constant _1271_MAGIC_VALUE = 0x1626ba7e;
-    IEntryPoint private immutable _entryPoint;
+    IEntryPoint private immutable _ENTRY_POINT;
     // keccak256("EIP712Domain(string name,string version,uint256 chainId,address verifyingContract)")
-    bytes32 private constant DOMAIN_SEPARATOR_TYPEHASH =
+    bytes32 private constant _DOMAIN_SEPARATOR_TYPEHASH =
         0x8b73c3c69bb8fe3d512ecc4cf759cc79239f7b179b0ffacaa9a75d522b39400f;
     // keccak256("LightAccountMessage(bytes message)");
-    bytes32 private constant LA_MSG_TYPEHASH = 0x5e3baca2936049843f06038876a12f03627b5edc98025751ecf2ac7562640199;
+    bytes32 private constant _LA_MSG_TYPEHASH = 0x5e3baca2936049843f06038876a12f03627b5edc98025751ecf2ac7562640199;
+    bytes32 private constant _NAME_HASH = keccak256(bytes("LightAccount"));
+    bytes32 private constant _VERSION_HASH = keccak256(bytes("1"));
```

Comment:
Nit: `keccak256(bytes("foo"))` yields the same result as `keccak256("foo")` and our comments above omit the `bytes` wrapper. But the ERC and OZ implementations follow this pattern so I think it's fine to keep it this way too.

Also, might be good to have consistent usage between precomputed values vs. inline expressions. It seems we can assign the commented expressions to `_DOMAIN_SEPARATOR_TYPEHASH` and `_LA_MSG_TYPEHASH` (they won't get recomputed at runtime). It actually might save gas on deployment too (double check me on this).